### PR TITLE
issue-25 fix JTA usage and dep scope.

### PR DESCRIPTION
**Pull request by tremes**
This change is [Reviewable](https://reviewable.io/reviews/arquillian/arquillian-container-weld/26)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -139,7 +139,6 @@
             <groupId>javax.transaction</groupId>
             <artifactId>jta</artifactId>
             <version>${jta.api.version}</version>
-            <scope>provided</scope>
```
Comment (johnament) [inline pom.xml]: Does this still need to be provided if you've fixed how transactions are registered?  Maybe optional would be a better version.

Comment (tremes) [inline pom.xml]: Sorry I don't follow. It's not provided anymore so do you suggest to introduce it with scope `compile` and as optional dep right?

Comment (johnament) [inline pom.xml]: Right.  You fixed the underlying issue (it was always adding JTA support).  Now that its only for web modules, bringing in JTA support always doesn't make sense.  Now it should either be provided or compile & optional.    The prior problem was that it was marked as provided, but was required always.

**Comment by tremes** [comment]
Fixed. Thanks @johnament

**Pull request closed without merging**